packages/ethereum-contracts/test/foundry/deployments/ForkBaseline.t.sol

```diff
@@ -0,0 +1,632 @@
+// SPDX-License-Identifier: AGPLv3
```

Comment:
Test vector management for future reference:

-  This is more like a smoke test.
-  Think of how to merge into ethereum-contracts/test/foundry.

Reply:
addressed: 36e0b2e361467d1b06d8d659ecd291f977c46d38